### DISCUSSION TITLE
Revit.Extensions is not compatible with net8.0?

### DISCUSSION THREAD

**Discussion by benkoshy** [Nice3point/RevitExtensions#6] [Q&A] [answered]
Hi there

First: thank you for your work on open source projects.

I am trying to use your extension library with net8.0 on a new revit command I am trying - a hello world revit command:

```sh
Package Nice3point.Revit.Extensions 2025.0.0 is not compatible with net8.0 (.NETCoreApp,Version=v8.0). 
Package Nice3point.Revit.Extensions 2025.0.0 supports: net8.0-windows7.0 (.NETCoreApp,Version=v8.0)
```

Was seeking a pointer on this issue?

I looks like it will only work on the previous version `net8.0-windows7.0 ` ?

Please let me know if there is anything I can do PR wise in order to make it work?

**Comment by Nice3point** [comment, marked as answer]
Hi, please read this article:
https://thebuildingcoder.typepad.com/blog/2024/04/migrating-from-net-48-to-net-core-8.html

You have to use `<TargetFramework>net8.0-windows</TargetFramework>` instead of `<TargetFramework>net8.0</TargetFramework>`. Revit API supports only Windows systems, there's no mistake, that's exactly how it's supposed to work
